microservice/challenge-1/app42/backend/server.ts

```diff
@@ -0,0 +1,43 @@
+import http = require("http");
+import log4js = require("log4js");
+
+import appManager = require("./src/AppManager");
```

Comment:
Some nodejs modules works well being imported with `require`, is there any reason your application files are not imported with `import { appManager } from './src/AppManager'` for example? What could be improved to respect some TS and ES6 inside your AppManager declaration?

Reply:
in the codebase, modules have single responsibility and they export one thing only which delivers the responsibility. For example, in case of appManager, the module exports a single instance of app and nothing else.

I hope I understood your point well and express mine well.

Reply:
Yeah, fair enough. Was just trying to figure out why in most usage, you would use `from` instead of `require`, but not here, where I can see AppManager exporting only the creation method you're using, so that you could have `import { createApp } from './src/AppManager'`.